examples/manifest.toml

```diff
@@ -6,105 +6,126 @@
-For the simplest possible examples showing how to use each api,
-check out [Types](/docs/reference/types).
+Examples related to robotics, autonomous systems, and interfacing with sensor hardware.
 """
 examples = [
-  "minimal",
-  "clock",
-  "eigen_opencv",
-  "multithreading",
-  "multiprocessing",
-  "plots",
+  "ros_node",
+  "nuscenes",
+  "kiss-icp",
+  "live_depth_sensor",
+  "lidar",
+  "rrt-star",
+  "rgbd",
+  "nv12",
 ]
 
-[categories.paper-visualizations]
+[categories.generative-vision]
 order = 3
-title = "Paper Visualizations"
+title = "Generative computer vision and LLMs"
 prelude = """
-The following examples use Rerun to create visual walkthroughs of papers. They are typically forks
-from the official open-source implementations adding Rerun as the visualizer.
-Check out the respective READMEs for installation instructions.
-For the simplest possible examples showing how to use each api,
-check out [Types](/docs/reference/types).
+Examples related to generative AI.
 """
 examples = [
-  "differentiable_blocks_world",
-  "limap",
+  "controlnet",
+  "depth_guided_stable_diffusion",
   "mcc",
   "shape_pointe",
-  "simplerecon",
-  "slahmr",
-  "tapir",
-  "widebaseline",
-  "arflow",
-  "kiss-icp",
+  "llm_embedding_ner",
 ]
 
-[categories.setup]
+[categories.video-image]
 order = 4
-title = "Setup"
+title = "Image and video understanding"
 prelude = """
-### Examples with Real / Artificial Data
-To run these examples, make sure you have the Rerun repository checked out
-and the latest SDK installed.
+Examples related to image and video processing, highlighting Rerun's 2D capabilities.
+"""
+examples = [
+  "detect_and_track_objects",
+  "face_tracking",
+  "human_pose_tracking",
+  "gesture_detection",
+  "live_camera_edge_detection",
+  "segment_anything_model",
+  "tapir",
+]
 
-```bash
-pip install --upgrade rerun-sdk  # install the latest Rerun SDK
-git clone git@github.com:rerun-io/rerun.git  # Clone the repository
-cd rerun
-git checkout latest  # Check out the commit matching the latest SDK release
-```
-> Note: Make sure your SDK version matches the examples.
-For example, if your SDK version is `0.3.1`, check out the matching tag
-in the Rerun repository by running `git checkout v0.3.1`.
+[categories.3d-reconstruction]
+order = 5
+title = "3D reconstruction and modelling"
+prelude = """
+SLAM, photogrammetry and other 3D modelling examples.
+"""
+examples = [
+  "structure_from_motion",
+  "open_photogrammetry_format",
+  "kiss-icp",
+  "differentiable_blocks_world",
+  "signed_distance_fields",
+  "raw_mesh",
+]
 
-### Paper Visualizations
-To reproduce the paper visualizations check out the README of the respective
-Rerun forks.
+[categories.integrations]
+order = 6
+title = "Integrations"
+prelude = """
+Integration with 3rd party tools, formats, libraries, and APIs.
 """
-examples = []
+examples = ["ros_node", "vrs", "eigen_opencv", "dicom_mri", "blueprint_stocks"]
```

Comment:
Add arflow
```suggestion
examples = [
    "ros_node",
    "vrs",
    "eigen_opencv",
    "dicom_mri",
    "blueprint_stocks",
    "arflow",
]
```

Reply:
Related, don't know if blueprint_stocks really fits here but maybe it's the best for now 🤷